src/ontology/mondo-edit.obo

```diff
@@ -19142,7 +19142,7 @@ is_a: MONDO:0037872 {source="MONDO:Redundant", source="linkedlifedata"} ! bordet
 intersection_of: MONDO:0005550 ! infectious disease
 intersection_of: disease_has_infectious_agent NCBITaxon:519 ! Bordetella parapertussis
 disjoint_from: MONDO:0005077 ! pertussis
-relationship: disease_has_feature HP:0031247 {severity="mild"} ! Whooping cough
+relationship: disease_has_feature HP:0031247 {comment="mild severity"} ! Whooping cough
```

Comment:
is this correct? I've never seen this before. Shouldn't this be a source to an xref?
cc @cmungall

Reply:
I just wanted to get rid of the severity property, is it is illegal! I dont think there is any other severity Modeling like this anywhere, so I turned it into a comment. Do you usually do severity with something like PATO?

Reply:
it should be 'has modifier' some Mild (mild from PATO, yes)

I fixed this on the PR